pkg/filesystem/local_directory_test.go

```diff
@@ -236,31 +258,38 @@ func TestLocalDirectoryOpenFileExistent(t *testing.T) {
 }
 
 func TestLocalDirectoryOpenFileNonExistent(t *testing.T) {
-	d := openTmpDir(t)
+	d, p := openTmpDir(t)
+	defer os.RemoveAll(p)
 	_, err := d.OpenFile("file", os.O_RDONLY, 0666)
 	require.True(t, os.IsNotExist(err))
 	require.NoError(t, d.Close())
 }
 
 func TestLocalDirectoryOpenFileSymlink(t *testing.T) {
-	d := openTmpDir(t)
+	if runtime.GOOS == "windows" {
```

Comment:
What's the reason this test won't work on Windows? Is there no way to prevent it from following junction points? If not, can we at least document this here?

Reply:
What is the test actually about on Linux? I lost the plot with `/etc/passwd` and `syscall.ELOOP`.

Reply:
That calling OpenFile() on a symlink doesn't actually follow the symlink. It should fail, as we shouldn't allow bb-worker to be tricked into follow them. Symlinks should be treated as ordinary objects in the file system.

`/etc/passwd` was chosen here as an example; any pathname string could be used here. I just thought that `/etc/passwd` was a good example of a file that should obviously not be accessible.